### PR TITLE
Document the Fluent Plugin

### PR DESCRIPTION
**Pull request by slavafomin**
Any suggestions are very welcome!

### CODE REVIEW

File: site/docs/plugins/i18n-fluent.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
+
+# Internationalization with Fluent
+
+[Fluent](https://projectfluent.org/) is a localization system for natural-sounding translations created and maintained by Mozilla Foundation.
```
Comment (roj1512) [inline site/docs/plugins/i18n-fluent.md:4]:
```suggestion
[Fluent](https://projectfluent.org/) is a localization system made by the Mozilla Foundation for natural-sounding translations.
```

File: site/docs/plugins/i18n-fluent.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+It has a very powerful and at the same time elegant syntax, that enables even non-developer people to efficiently work with translation files.
```
Comment (roj1512) [inline site/docs/plugins/i18n-fluent.md:5]:
```suggestion
It has a very powerful and elegant syntax that lets anyone write efficient and fully-understandable translations.
```

File: site/docs/plugins/i18n-fluent.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+Thanks to the [grammY Fluent integration plugin](https://github.com/grammyjs/fluent) you can easily translate your bot to multiple languages using industry best practices like variable substitution, pluralization, locale negotiation, conditional messages and message reuse.
```
Comment (roj1512) [inline site/docs/plugins/i18n-fluent.md:6]:
```suggestion
This plugin takes advantage of this amazing localization system to make grammY-powered bots fluent with high-quality translations.
```

**Review by roj1512 — CHANGES_REQUESTED** [review]

**Review by roj1512 — COMMENTED** [review]
The file might be renamed to `fluent.md`. The `## Install` section might be replaced with a "Plugin Summary" section with links to the repository.

File: site/docs/plugins/i18n-fluent.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+  // you can specify multiple locales if you want
+  locales: 'en',
+  
+  // You can specufy the translation content directly:
+  source: '{YOUR TRANSLATION FILE CONTENT}',
+
+  // OR you can specify a path to translation file or multiple files:
+  filePath: [
+    `${__dirname}/feature-1/translation.en.ftl`,
+    `${__dirname}/feature-2/translation.en.ftl`
```
Comment (KnorpelSenf) [inline site/docs/plugins/i18n-fluent.md:46]: Does this work cross-platform?

File: site/docs/plugins/i18n-fluent.md [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+    applesCount: 1,
+  }));
+
+});
+```
+
+Now, you can start your bot and use the `/i18n_test` command.
+It should render the following message:
+
+```text
```
Comment (KnorpelSenf) [inline site/docs/plugins/i18n-fluent.md:153]:
```suggestion
```text:no-line-numbers
```

File: site/docs/plugins/i18n-fluent.md (hunk 4 above)
Comment (KnorpelSenf) [reply site/docs/plugins/i18n-fluent.md:46]: To be clear, I am referring to file paths on Windows. I do not mean Deno & Node.

Comment (slavafomin) [reply site/docs/plugins/i18n-fluent.md:46]: All the examples are meant for Node.js and Linux. I don't think that we should complicate the example code to support Windows. The Windows devs should know what to do with this.

**Comment by KnorpelSenf** [comment]
Any updates on this?

File: site/docs/plugins/fluent.md [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+[Fluent](https://projectfluent.org/) is a localization system made by the Mozilla Foundation for natural-sounding translations.
+It has a very powerful and elegant syntax that lets anyone write efficient and fully-understandable translations.
+This plugin takes advantage of this amazing localization system to make grammY-powered bots fluent with high-quality translations.
+
+## Instantiate Fluent
+
+The next step would be to create a Fluent instance:
+
+```typescript
+import { Fluent } from "@moebius/fluent";
```
Comment (KnorpelSenf) [inline site/docs/plugins/fluent.md:12]: This is outdated, no?

Comment (roj1512) [reply site/docs/plugins/fluent.md:12]: No it is not. The `Fluent` class is from the `@moebius/fluent` package.

Comment (KnorpelSenf) [reply site/docs/plugins/fluent.md:12]: Ohhh I see, my bad.

**Review by KnorpelSenf — APPROVED** [review]
Rubber-stamp LGTM.

**Comment by roj1512** [comment]
Omg no

**Comment by roj1512** [comment]
What the hell did I do

**Pull request merged**